sonar-ws/src/main/java/org/sonarqube/ws/client/measure/MeasuresService.java

```diff
@@ -78,8 +80,9 @@ public ComponentTreeWsResponse componentTree(ComponentTreeWsRequest request) {
 
   public ComponentWsResponse component(ComponentWsRequest request) {
     GetRequest getRequest = new GetRequest(path(ACTION_COMPONENT))
-      .setParam(PARAM_COMPONENT_ID, request.getComponentId())
-      .setParam(PARAM_COMPONENT_KEY, request.getComponentKey())
+      .setParam(DEPRECATED_PARAM_COMPONENT_ID, request.getComponentId())
+      .setParam(DEPRECATED_PARAM_COMPONENT_KEY, request.getComponentKey())
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "getComponentKey"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)